doc/man1/enc.pod

```diff
@@ -199,12 +199,12 @@ encrypting a file and read from the encrypted file when it is decrypted.
 
 Some of the ciphers do not have large keys and others have security
 implications if not used correctly. A beginner is advised to just use
-a strong block cipher in CBC mode such as bf or des3.
+a strong block cipher in CTR mode such as AES or Camellia.
```

Comment:
Why the CBC to CTR change? Also not sure about the recommendation to use Camellia. Perhaps just leave it at AES.